src/main/java/com/github/ibm/mapepire/requests/PreparedExecute.java

```diff
@@ -21,35 +22,31 @@ public PreparedExecute(final DataStreamProcessor _io, final JsonObject _reqObj,
 
     @Override
     protected void go() throws Exception {
-        JsonElement parms = super.getRequestField("parameters");
-        boolean isBatch = super.getRequestFieldBoolean("batch", false);
+        JsonArray parms = super.getRequestField("parameters").getAsJsonArray();
+        boolean isBatch = !parms.isEmpty() && parms.get(0).isJsonArray();
```

Comment:
With this change, it looks like we no longer have a `batch` option on the request and instead determine it based on if the parameter is an array of arrays. The README will need to be updated to reflect this which has some references to this option on `execute` request type.

Reply:
The purpose of the batch option (at least during lackadaisical code bringup) was to allow clients to add things to batch explicitly and in multiple calls before executing the batch statement. 

We can't break that design pattern because it will be needed for the future JDBC driver. Reading through the new code, I believe this behavior is unchanged. I would ask for a test to support this, but I don't believe the js API allows for a batch query without immediately running. 

Does the current js client send the `execute` with parameters, and then run request in a subsequent datastream flow? (I presume so)

Reply:
@ThePrez Good point. In the current code, the design is broken as batch operations are always executed immediately. I will fix this by adding an option to just add to batch without executing immediately.

Reply:
Side note: if there are behaviour changes to `batch`, please update the readme docs. Find `batch` in the table to see where it is used: https://github.com/Mapepire-IBMi/mapepire-server?tab=readme-ov-file#usage

Reply:
I think for now we will allow the client to batch sql statements together just on the client side. I already have very simple code in the js client to do it.